src/pfs/pfs.proto

```diff
@@ -14,11 +14,13 @@ import "auth/auth.proto";
 ////  PFS Data structures (stored in etcd)
 
 message Repo {
+  option (gogoproto.goproto_stringer) = false;
```

Comment:
the other options we use put these in `[]` - do we know why that is?  maybe just do the same thing here for consistency?

Reply:
I think that might be special syntax for the ones that apply to specific fields, but I can try to figure that out

Reply:
Yeah, seems to just be for tags